.github/workflows/wcag_test.yml

```diff
@@ -0,0 +1,14 @@
+name: WCAG tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  call_wcag_test:
+    uses: yext/slapshot-reusable-workflows/.github/workflows/wcag_test.yml@v1
+    secrets:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+    with:
+      build_script: 'npm run storybook & sleep 10 && curl http://localhost:6006 -I'
```

Comment:
I'm sure you looked into this but can we avoid the `sleep 10`? Maybe some option in start-storybook?

Reply:
Yeah, I tried finding a way to not use `sleep` and checked again in the `start-storybook` options. The closest option to what we want is --smoke-test, but that stops serving the site once it successfully starts. We need the site to be running when the `test-storybook` command is run. The only other way I saw to do this as part of a GH workflow was either to host the site somewhere public, or involved setting up [service containers](https://docs.github.com/en/actions/using-containerized-services/about-service-containers). Service containers seemed like they might be overkill, but I can look into it more

Reply:
is something like this possible? I don't have any experience trying stuff like this https://stackoverflow.com/questions/21475639/wait-until-service-starts-in-bash-script

Reply:
sure! I updated it to be a while loop instead so it keeps checking every second, instead of having an arbitrary sleep time of 10 seconds

Reply:
nice!